### ISSUE TITLE
Design Spec

### ISSUE THREAD

**Issue by josephgupkim** [mayankdwivedidev/prj-rev-bwfs-dasmoto#3]
Remember that not everyone has access to this file path:
"/Users/dv/Desktop/Dasmoto"

Therefore you'll need to use relative paths instead of absolute paths so that your project behaves the same no matter where the project is opened. For example:
"./Resources/images/hacksaw.jpeg"

Same for the background url in your stylesheet:
"background-image:url("../Resources/images/pattern.jpeg");"

After making these fixes, I can confidently say your project looks identical to the design spec. Great job!

**Comment by mayankdwivedidev** [comment]
Yeah! Thank you for pointing this out. Again, I realised the mistake as soon as I submitted the project. Will be more pro-active going forward. Thank you :-)
